graylog2-web-interface/src/components/common/DatePicker.tsx

```diff
@@ -16,7 +16,7 @@
  */
 import PropTypes from 'prop-types';
 import React from 'react';
-import DayPicker from 'react-day-picker';
+import DayPicker, { DayModifiers } from 'react-day-picker';
```

Comment:
Do we want to do explicit type imports?

```suggestion
import DayPicker from 'react-day-picker';
import type { DayModifiers } from 'react-day-picker';
```

Reply:
In this case we are not gaining anything, because the value-level export is used too. So, a general suggestion is to prefer type-only imports when a module is imported only for a type, if not it's up to the author.